commands/createfixpullrequests_test.go

```diff
@@ -108,72 +53,6 @@ var testPackagesData = []struct {
 	},
 }
 
-func getGenericFixPackageVersionFunc() FixPackagesTestFunc {
-	return func(test packageFixTest) CreateFixPullRequestsCmd {
-		return CreateFixPullRequestsCmd{
-			details: &utils.ScanDetails{
-				Project: &utils.Project{
-					PipRequirementsFile: test.packageDescriptor,
-					WorkingDirs:         []string{test.testPath},
-				},
-			},
-		}
-	}
-}
-
-func getMavenFixPackageVersionFunc() func(test packageFixTest) CreateFixPullRequestsCmd {
-	return func(test packageFixTest) CreateFixPullRequestsCmd {
-		mavenDepToPropertyMap := map[string][]string{
-			test.impactedPackaged: {"junit:junit", "3.8.1"},
-		}
-		cfp := CreateFixPullRequestsCmd{
-			mavenDepToPropertyMap: mavenDepToPropertyMap,
-		}
-		return cfp
-	}
-}
-
-func TestFixPackageVersion(t *testing.T) {
-	currentDir, testdataDir := getTestDataDir(t)
-	defer func() {
-		assert.NoError(t, os.Chdir(currentDir))
-	}()
-
-	for _, test := range packageFixTests {
-		func() {
-			// Create temp technology project
-			projectPath := filepath.Join(testdataDir, test.technology.ToString())
-			tmpProjectPath, cleanup := testdatautils.CreateTestProject(t, projectPath)
-			defer cleanup()
-			test.testPath = tmpProjectPath
-			assert.NoError(t, os.Chdir(tmpProjectPath))
-
-			t.Run(test.technology.ToString(), func(t *testing.T) {
-				cfg := test.fixPackageVersionCmd(test)
-				// Fix impacted package for each technology
-				fixVersionInfo := utils.NewFixVersionInfo(test.fixVersion, test.technology, true)
-				assert.NoError(t, cfg.updatePackageToFixedVersion(test.impactedPackaged, fixVersionInfo))
-				file, err := os.ReadFile(test.packageDescriptor)
-				assert.NoError(t, err)
-				if test.shouldNotFix {
-					assert.NotContains(t, string(file), test.fixVersion)
-				} else {
-					assert.Contains(t, string(file), test.fixVersion)
-					// Verify that case-sensitive packages in python are lowered
-					assert.Contains(t, string(file), strings.ToLower(test.impactedPackaged))
-				}
-			})
-		}()
-	}
-}
-func getTestDataDir(t *testing.T) (string, string) {
-	currentDir, err := os.Getwd()
-	assert.NoError(t, err)
-	testdataDir, err := filepath.Abs("testdata/projects")
-	assert.NoError(t, err)
-	return currentDir, testdataDir
-}
-
```

Comment:
After this removal, are we testing the same logic for all these package managers in other tests?

Reply:
Yeah it just moved to every package handler